validator/client/src/main/java/tech/pegasys/teku/validator/client/Graffiti.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.client;
+
+import java.nio.file.Path;
+import java.util.Optional;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.teku.validator.client.loader.graffiti.GraffitiLoaderException;
+import tech.pegasys.teku.validator.client.loader.graffiti.GraffitiParser;
+
+/**
+ * A type to hold the current graffiti value. Graffiti can be loaded from 2 places:
+ *
+ * <ul>
+ *   <li>The config file at initialisation
+ *   <li>The graffiti file which will be regularly checked
+ * </ul>
+ *
+ * In the case that the file cannot be read, the graffiti value will remain unchanged from its
+ * previous value.
+ */
+public class Graffiti {
```

Comment:
Let's add some tests for this class.  A few cases I can think of: 
* graffiti and graffitiFile are empty / null
* graffiti is empty / null
* graffitiFile is empty / null
* graffitiFile is set, but the actual file doesn't exist
* read from file successfully, remove file from disk, read again

Reply:
Will come back to this once we're settled on the structure. Totally agree.

Reply:
Done